### PR TITLE
fix(surveys): Fix survey prop naming

### PR DESCRIPTION
**Pull request by liyiy**
## Problem

dismiss -> dismissed

https://github.com/PostHog/posthog-js/pull/861
## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/lib/taxonomy.tsx [hunk 1]
```diff
@@ -826,6 +826,14 @@ export function getKeyMapping(
                 description: `The response value for the ${index}${suffix} question in the survey.`,
             }
         }
+    } else if (value.startsWith('$survey_responded/')) {
```
Comment (neilkakkar) [inline frontend/src/lib/taxonomy.tsx:829]: this one's already on line 800, just edit that?

Comment (liyiy) [reply frontend/src/lib/taxonomy.tsx:829]: isn't that for the event property `survey_response`?

Comment (liyiy) [reply frontend/src/lib/taxonomy.tsx:829]: oh no wait it has a slash in it. I assume `survey_response_` handles `survey_response` from the event property?

Comment (liyiy) [reply frontend/src/lib/taxonomy.tsx:829]: Yeah I think I added it a while ago LOL nvm

Comment (neilkakkar) [reply frontend/src/lib/taxonomy.tsx]: There's a separate one in KEY_MAPPINGS for the response. These ones are needed only when it's a programmatic property, i.e some part changes, like the id

**Pull request merged**